src/unit/test_hashtable.c

```diff
@@ -803,6 +803,8 @@ int test_random_entry_with_long_chain(int argc, char **argv, int flags) {
     UNUSED(argv);
     UNUSED(flags);
 
+    hashtableSetResizePolicy(HASHTABLE_RESIZE_FORBID);
```

Comment:
How will this test case work if it's not possible to resize the table? It's always possible to create the first table, which will be just one bucket, but then it can't resize for the rest of this test case, so all elements will be in one bucket chain...?

Sure, it will give you a long bucket chain, bit it's not exactly the idea of this test case. The idea is that you have a skewed hashtable where many entries just ended up in the same bucket chain and others are almost empty.

@SoftlyRaining You wrote this test test originally. Would you like to help out?